lib/SoftwareTree.py

```diff
@@ -1,301 +1,213 @@
-                layout['cells'] = [[0, 0, 1, len(layout['rows']) - 1]] + [
-                    [cell[0] + 1, cell[1], cell[2] + 1, cell[3]] for cell in layout['cells']
-                ]
-                window.set_layout(layout)
-                for view in window.views():
-                    (group, index) = window.get_view_index(view)
-                    window.set_view_index(view, group + 1, index)
+                # Evenly space the original views
+                tree_view_width = min(0.25, layout['cols'][1] / 2.0)
+                new_orig_views = list(map(lambda x: x + (1 - x) * tree_view_width, layout['cols'][1:-1])) 
+                layout['cols'] = [0, tree_view_width] + new_orig_views + [layout['cols'][-1]]
+
+            layout['cells'] = [[0, 0, 1, len(layout['rows']) - 1]] + [
+                [cell[0] + 1, cell[1], cell[2] + 1, cell[3]] for cell in layout['cells']
+            ]
+            window.set_layout(layout)
 
             tree_view = window.new_file()
             tree_view.settings().set('line_numbers', False)
             tree_view.settings().set('gutter', False)
             tree_view.settings().set('rulers', [])
             tree_view.set_read_only(True)
-            tree_view.set_name('Code Time Tree View')
+            tree_view.set_name(CODETIME_TREEVIEW_NAME)
             tree_view.set_scratch(True)
-            if window.num_groups() > 1: 
-                (group, index) = window.get_view_index(orig_view)
-                if group != 0:
-                    group = 0
-                    window.set_view_index(tree_view, group, 0)
-                    window.focus_view(orig_view) # Um... Seems like a bug
-        else:
-        	tree_view.erase_phantoms('remote_tree')
+            if window.num_groups() > 1:
+                for view in window.views():
+                    (group, index) = window.get_view_index(view)
+                    window.set_view_index(view, group + 1, 0)
+                window.set_view_index(tree_view, 0, 0)
+                window.focus_view(orig_view)
+
+        self.phantom_set = sublime.PhantomSet(tree_view, 'software_tree')
 
-        self.view = tree_view
-        self.phantom_set = sublime.PhantomSet(tree_view, 'remote_tree')
+        if len(window.views()) == 1:
+            window.focus_group(1)
 
-        # trees.append(self) 
+        statusBarMessage = 'Show status bar metrics' if getValue("show_code_time_status", True) else 'Hide status bar metrics'
 
         self.tree = { 
         	'index': 0,
         	'depth': 0,
         	'path': '',
         	'name': '',
+            'id': '',
         	'dir': True,
         	'expanded': False,
         	'childs': [
                 {
-                    'index': 1,
                     'depth': 1,
+                    'id': 'code-time-actions',
                     'name': 'CODE TIME',
                     'dir': True,
                     'expanded': False,
                     'childs': [
```

Comment:
Could create a separate model for `childs` that can be imported here or move to a function and put at the bottom of the file